code/w2v_glove.py

```diff
@@ -3,13 +3,15 @@
 from gensim.scripts.glove2word2vec import glove2word2vec
 from gensim.models import KeyedVectors
 from gensim.models import Word2Vec
+import constants
 
 from w2v import W2V
 
+WORD_CORPUS_SIZE = '840B'
```

Comment:
This should be pulled into constants.py
Perhaps programmatically get the corpus size corresponding to the dimension?